src/generate_initial_weekly_plots.R

```diff
@@ -111,12 +134,11 @@ generate_initial_weekly_plots <- function(all_df_list, pending_df_list, site_arg
             theme_bw() +
             scale_x_datetime(date_breaks = "1 day",
                              date_labels = "%b %d",
-                             minor_breaks = date_seq,
-                             sec.axis = sec_axis(~ ., breaks = date_seq, labels = unique(week_plot_data$weekday))) +
+                             minor_breaks = date_seq) +
             theme(legend.position = 'bottom',
                   panel.grid.major = element_blank(),
                   panel.grid.minor = element_blank()) +
-            # annotate("text", x = date_seq, y = min(week_plot_data$mean, na.rm = TRUE) - 1, label = unique(week_plot_data$weekday), hjust = 0) +
+            annotate("text", x = date_seq, y = min(week_plot_data$mean, na.rm = TRUE) - 1, label = unique(week_plot_data$weekday), hjust = 0) +
```

Comment:
Question: are these plotting updates (L104-141) intentional? 

The updates I made to make sure that the days are accurate and to prevent the annotations from warping the plot seem to be in an older state from the recent updates. Tagging this in case it was unintentional.

Reply:
Yes this was unintentional!